distribution/asf-release-process-guide.md

```diff
@@ -71,6 +71,17 @@ Finally, the sample [`docker-compose.yml`](https://github.com/apache/druid/blob/
 
 Once this is completed, open a PR to the master branch. Also, be sure to confirm that these versions are all correct in the release branch, otherwise fix them and open a backport PR to the release branch.
 
+#### Version updates on master and release branch
```

Comment:
I think these can all be done in the master branch and be part of the previous section, maybe instead add a bit that if any of these steps is missed, then update it in master and backport to the release branch?